<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApproachRetract</class>
 <widget class="QWidget" name="ApproachRetract">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1100</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Force-Distance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QWidget" name="upper_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="WidgetPlotFD" name="widget_plot_fd"/>
        <widget class="WidgetPlotPreproc" name="widget_plot_preproc"/>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>545</width>
          <height>800</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <widget class="TabPreprocess" name="tab_preprocess">
         <attribute name="title">
          <string>Preprocess</string>
         </attribute>
         <attribute name="toolTip">
          <string>curve preprocessing</string>
         </attribute>
        </widget>
        <widget class="TabFit" name="tab_fit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="title">
          <string>Fit</string>
         </attribute>
         <attribute name="toolTip">
          <string>fitting parameters</string>
         </attribute>
        </widget>
        <widget class="TabEdelta" name="tab_edelta">
         <attribute name="title">
          <string>E(Î´)</string>
         </attribute>
         <attribute name="toolTip">
          <string>elastic modulus vs. indentation depth</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_plot">
         <attribute name="title">
          <string>Plot</string>
         </attribute>
         <attribute name="toolTip">
          <string>plotting parameters</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="plot_prefs" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QCheckBox" name="cb_mpl_rescale_plot_x">
               <property name="text">
                <string>Autoscale X-axis to fitting range</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="cb_mpl_rescale_plot_x_min">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="minimum">
                  <double>-100.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>-5.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_plot_xax_min">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>X-axis minimum</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_plot_xax_max">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>X-axis maximum</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="cb_mpl_rescale_plot_x_max">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <double>-100.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>2.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="cb_mpl_rescale_plot_y">
               <property name="text">
                <string>Autoscale Y-axis to fitting range</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_5">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item row="1" column="0">
                <widget class="QLabel" name="label_plot_yax_min">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Y-axis maximum</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_plot_yax_max">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Y-axis minimum</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDoubleSpinBox" name="cb_mpl_rescale_plot_y_min">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <double>-100.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>-0.500000000000000</double>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QDoubleSpinBox" name="cb_mpl_rescale_plot_y_max">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <double>-100.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>100.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>7.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="TabInfo" name="tab_info">
         <attribute name="title">
          <string>Info</string>
         </attribute>
         <attribute name="toolTip">
          <string>cuve information</string>
         </attribute>
        </widget>
        <widget class="TabQMap" name="tab_qmap">
         <attribute name="title">
          <string>QMap</string>
         </attribute>
         <attribute name="toolTip">
          <string>quantitative imaging</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="lower_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="list_curves">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <column>
         <property name="text">
          <string>file</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>enum</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>rating</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>use</string>
         </property>
        </column>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QWidget" name="widget_list_controls" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_fitall">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Apply Model and Fit All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Curve rating</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Scheme</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cb_rating_scheme">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>Default (zef18 &amp; Extra Trees)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Disabled</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QDoubleSpinBox" name="sp_rating_thresh">
                <property name="decimals">
                 <number>1</number>
                </property>
                <property name="minimum">
                 <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>10.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>5.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_rating_filter">
                <property name="text">
                 <string>Apply Threshold</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_autosave">
           <property name="text">
            <string>Autosave fit results as .tsv</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>i&amp;n leaf directories</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>somep&amp;lace else</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_rater">
           <property name="text">
            <string>Start Curve Rater</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabPreprocess</class>
   <extends>QWidget</extends>
   <header>pyjibe.fd.tab_preprocess</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabQMap</class>
   <extends>QWidget</extends>
   <header>pyjibe.fd.tab_qmap</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabFit</class>
   <extends>QWidget</extends>
   <header>pyjibe.fd.tab_fit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabEdelta</class>
   <extends>QWidget</extends>
   <header>pyjibe.fd.tab_edelta</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabInfo</class>
   <extends>QWidget</extends>
   <header>pyjibe.fd.tab_info</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPlotFD</class>
   <extends>QWidget</extends>
   <header>pyjibe.fd.widget_plot_fd</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPlotPreproc</class>
   <extends>QWidget</extends>
   <header>pyjibe.fd.widget_plot_preproc</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cb_mpl_rescale_plot_x</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_mpl_rescale_plot_x_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>870</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_mpl_rescale_plot_x</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_mpl_rescale_plot_x_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>870</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_mpl_rescale_plot_x</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_plot_xax_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_mpl_rescale_plot_x</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_plot_xax_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_mpl_rescale_plot_y</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_mpl_rescale_plot_y_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>870</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_mpl_rescale_plot_y</sender>
   <signal>toggled(bool)</signal>
   <receiver>cb_mpl_rescale_plot_y_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>870</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_mpl_rescale_plot_y</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_plot_yax_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_mpl_rescale_plot_y</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_plot_yax_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
